nav2_robot/include/nav2_robot/robot.hpp

```diff
@@ -38,6 +38,7 @@ class Robot
   bool getOdometry(nav_msgs::msg::Odometry::SharedPtr & robot_odom);
   std::string getName();
   void sendVelocity(geometry_msgs::msg::Twist twist);
+  double getWidth() {return width_;}
```

Comment:
This should be more like getFootprint or at least getRadius, however, I think this needs to be reconciled with the getFootprint on costmap. I'd rather you just got the footprint off the costmap until such time as we can remove the code on costmap and put a fully functional version here.

Reply:
I'll change it to `getRadius`. Getting the footprint from costmap will require adding a service.

Reply:
And costmap should really be getting the footprint from the robot class -  #563